Form/Type/StaticContentType.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ContentBundle\Form\Type;
+
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
+use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContent;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class StaticContentType extends AbstractType
+{
+    /**
+     * Builds the form.
+     *
+     * @see FormTypeInterface::buildForm()
+     *
+     * Available options:
+     *     - use_ivory_ckeditor bool use CKEditorType instead of TextareaType for body
+     *     - ivory_ckeditor_options array CKEditorType options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('title', TextType::class)
+            ->add(
+                'body',
+                $options['use_ivory_ckeditor'] ? CKEditorType::class : TextareaType::class,
+                $options['ivory_ckeditor_options']
```

Comment:
if we can simplify this, that would be great. FormExtension sounds like
it could be more flexible and open the full power to the user instead of
us trying to forward some flags to another part...

how would the admin look if we use a FormExtension? (or more important:
would it make it harder for the user to set up ivory?)